src/control/SConscript

```diff
@@ -147,9 +147,15 @@ def scons():
     gosrc = Dir('.').srcnode().abspath
     cartpath = Dir('#/src/cart/src/include').srcnode().abspath
 
+    prereqs.require(denv, 'ofi', 'hwloc')
+
     denv.AppendENVPath("CGO_LDFLAGS",
-                       denv.subst("-L$BUILD_DIR/src/cart/src/gurt"), sep=" ")
-    denv.AppendENVPath("CGO_CFLAGS", "-I%s" % cartpath, sep=" ")
+                       denv.subst("$_LIBDIRFLAGS "
```

Comment:
why the split string here?  Does it go over 80 characters if we just do "$_LIBDIRFLAGS $_RPATH"

Reply:
scons is not able to resolve the $ vars automatically.  It needs the denv.AppendENVPath().  Putting the whole string on  one line does go over 80 chars.  There current string formatting (where sep=" " lands) had precedent in the file, so I followed that existing style.